### PR TITLE
Feature/pin applet inside folder

### PR DESCRIPTION
**Pull request by nmialik**

### CODE REVIEW

File: src/components/Tables/AppletsTable/Table/AppletItem/AppletItem.styles.ts [hunk 1]
```diff
@@ -1,9 +1,38 @@
...
   align-items: center;
   margin-left: ${({ applet }: { applet: FolderApplet }) =>
     applet?.depth ? `${applet.depth * 4.4}rem` : 0};
 `;
+
+export const StyledPin = styled(Button)`
+  width: 4rem;
+  height: 4rem;
+  min-width: 4rem;
+  padding: 0;
```
Comment (kdanilenka32) [inline src/components/Tables/AppletsTable/Table/AppletItem/AppletItem.styles.ts:18]: Do we have Button without padding component?

Comment (nmialik) [reply src/components/Tables/AppletsTable/Table/AppletItem/AppletItem.styles.ts:18]: Replaced to StyledClearedButton

**Review by kdanilenka32 — APPROVED** [review]

File: src/svgSprite.ts [hunk 2]
```diff
@@ -6,18 +6,29 @@ export const svgSprite = (): string => `
...
   </symbol>
 
+  <symbol viewBox="0 0 24 24" id="svg-pin-outlined">
```
Comment (IhorNesterenko99) [inline src/svgSprite.ts:9]: We already have svg-pin and svg-pin-outlined. Could you pls check their combability and remove/replace them with new one?

Comment (nmialik) [reply src/svgSprite.ts:9]: fixed

**Review by IhorNesterenko99 — APPROVED** [review]

**Pull request merged**